# Fast Accurate
string fringe_coding_mode
---
int32 error_code
string error_description